tests/test_frame.py

```diff
@@ -560,10 +559,37 @@ def test_types_groupby() -> None:
     df8: pd.DataFrame = df.groupby("col1").transform("sum")
     s1: pd.Series = df.set_index("col1")["col2"]
     s2: pd.Series = s1.groupby("col1").transform("sum")
-    s3: pd.Series = df.groupby("col1")["col3"].agg(min)
-    df9: pd.DataFrame = df.groupby("col1")["col3"].agg([min, max])
-    df10: pd.DataFrame = df.groupby("col1").agg(
-        new_col=pd.NamedAgg(column="col2", aggfunc="max")
```

Comment:
I think the test for `df10` got lost

Reply:
It moved to `test_types_groupby_agg` and has a return type checking now.

Reply:
OK, now I see that.  Thanks.